solvable_files.sh

```diff
@@ -110,7 +110,7 @@ function correct_mtime() {
 
 	echo "$filepath"
 
-	if [ "$action" == "fix" ]
+	if [ "$action" == "fix" ] && [ -e "$filepath" ]
```

Comment:
you probably need to also add the same check on https://github.com/nextcloud-gmbh/mtime_fixer_tool_kit/pull/30/files#diff-0ee24123dd7eff187910dbfc5b7ae59536073b97c80ad2a289c26c3003467108R24

otherwise we could still try to access a file that is no longer existing
should make sure that whatever option are given to the script we do not trigger errors